lib/newman_conway.py

```
@@ -4,7 +4,24 @@
 # Space Complexity: ?
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """
-    pass
+    if num == 0:
+        raise ValueError("Input must be greater than 0.")
```

We should raise this error for _any_ value below the valid starting point of the sequence:
```py
    if num <= 0:
        raise ValueError("Input must be greater than 0.")
```